frontend/src/index.css

```diff
@@ -50,3 +50,25 @@ html {
   top: 16.5px;
   z-index: 100;
 }
+
+/* My Classes */
+
+.header-w {
+  font-style: normal;
+  line-height: 48px;
+  /* identical to box height */
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  color: #ffffff;
+}
+
+.corner {
```

Comment:
Just wondering, what are these styles here for?

Reply:
These are for courses but I was focused more on getting the endpoints to work. I'm going to use them later to style.